runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -162,42 +170,53 @@ def model(self, id: str) -> prophet.forecaster.Prophet:
         from prophet.serialize import model_from_json
         return model_from_json(self._model_json[id])
 
-    def _make_future_dataframe(self, id: str, horizon: int) -> pd.DataFrame:
+    def _make_future_dataframe(self, id: str, horizon: int, include_history: bool = True) -> pd.DataFrame:
         """
         Generate future dataframe for one model by calling the API from prophet
         :param id: identity for the Prophet model
         :param horizon: Int number of periods to forecast forward
+        :param include_history: Boolean to include the historical dates in the data
+            frame for predictions.
         :return: pd.Dataframe that extends forward from the end of self.history for the
         requested number of periods.
         """
-        start_time = self._timeseries_starts[id]
         end_time = pd.Timestamp(self._timeseries_end)
+        if include_history:
+            start_time = self._timeseries_starts[id]
+        else:
+            start_time = end_time + pd.Timedelta(value=1, unit=self._frequency)
```

Comment:
Why do we use a same `end_time` for all time series in prophet? It doesn't sound quite accurate.